### PR TITLE
Feature/upgrade flutter version

### PR DESCRIPTION
**Pull request by lakpa**

### CODE REVIEW

File: android/src/main/kotlin/com/ovatu/zettle/ZettlePlugin.kt [hunk 1]
```diff
@@ -313,6 +313,7 @@ class ZettlePlugin: FlutterPlugin, MethodCallHandler, ActivityAware, PluginRegis
                       "status" to "failed",
               )
             }
+            else -> {}
```
Comment (karspv) [inline android/src/main/kotlin/com/ovatu/zettle/ZettlePlugin.kt:316]: do we need this?

File: pubspec.yaml [hunk 2]
```diff
@@ -5,8 +5,7 @@ repository: https://github.com/ovatu/zettle_flutter
 homepage: https://ovatu.com/us
 
 environment:
-  sdk: ">=2.16.2 <3.0.0"
-  flutter: ">=2.5.0"
+  sdk: ">=2.16.0 <3.0.0"
```
Comment (karspv) [inline pubspec.yaml:8]: shouldn't this be increased also?

**Review by karspv — APPROVED** [review]
couple questions

File: android/src/main/kotlin/com/ovatu/zettle/ZettlePlugin.kt (hunk 1 above)
Comment (lakpa) [reply android/src/main/kotlin/com/ovatu/zettle/ZettlePlugin.kt:316]: yes, this what the latest flutter version is complaining about of switch statement being not exhaustive

File: pubspec.yaml (hunk 2 above)
Comment (lakpa) [reply pubspec.yaml:8]: no, it doesn't need to be

**Review by karspv — APPROVED** [review]

**Review by karspv — APPROVED** [review]

**Pull request merged**